app/models/prescription_drug.rb

```diff
@@ -39,4 +41,28 @@ def anonymized_data
      medicine_name: name,
      dosage: dosage}
   end
+
+  private
+
+  # this code should only be temporary, as it's part of monitoring a one-off experiment
+  # https://app.clubhouse.io/simpledotorg/story/3642/remove-unnecessary-medication-reminder-code
+  def log_medication_reminder_success
```

Comment:
This is not my finest code and I'm glad it's temporary. Hopefully it accomplishes our goal here.

Reply:
If we don't want to rely on callbacks, a couple of other ways: 
- Log this in the prescription drug sync_from_user controller
- Run a nightly job that collects all prescription drugs synced in the day, and logs this 

I am leaning towards option 2 since it'll be isolated.